regcomp_internal.h

```diff
@@ -919,7 +918,6 @@ static const scan_data_t zero_scan_data = {
  * Calls SAVEDESTRUCTOR_X if needed, then Simple_vFAIL()
```

Comment:
This comment is no longer accurate because the macro no longer calls `SAVEDESTRUCTOR_X`. Same for all the following `vFAIL` variants.